app/src/main/java/woowacourse/shopping/CustomBindingAdapter.kt

```diff
@@ -23,9 +23,9 @@ object CustomBindingAdapter {
         fun onCountChange(productId: Int, count: Int)
     }
 
-    @BindingAdapter("listener", requireAll = true)
+    @BindingAdapter("productCounterListener", requireAll = true)
```

Comment:
명시적 네이밍 변경 👍 
참고로 저는 이 바인딩 어댑터가 개발자가 임의로 생성한 커스텀 어댑터임을 강조하기 위해, 또 코드 네임스페이스를 관리하기 위해
```suggestion
    @BindingAdapter("bind_productCounterListener", requireAll = true)
```
위와 같이 bind prefix를 붙이기도 합니다. 이렇게 작성하면 bind로 시작하는 네임스페이스에서 한눈에 볼 수 있어요.
<img width="438" alt="image" src="https://github.com/woowacourse/android-shopping-cart/assets/112997521/baa54044-c066-4f49-85ff-3c83e9ac6e07">


이 스타일은 참고만 해주세요 🙂